doc/api/swagger.json

```diff
@@ -4363,6 +4363,216 @@
           }
         }
       }
+    },
+    "/tags": {
+      "get": {
+        "description": "Returns tags visible to current user according to given parameters",
+        "tags": [
+          "tags"
+        ],
+        "security": [
+          {
+            "basicAuth": []
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Tags returned",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "total": {
```

Comment:
I add it to be consistent with others API. But for configuration API may we can decide to use a simpler model (direct list of results)